docs/iris/src/whitepapers/change_management.rst

```diff
@@ -0,0 +1,393 @@
+effectively a change to the 'default behaviour' of a specific usage.  Although
+this seems similar to adding a keyword, the cases where the new behaviour
+operates and where it does not are not distinguishable by a different code
+syntax, which makes this somewhat dangerous.  We do use this type of change,
+but any behaviour 'mode' controls of this sort are usually added as part of the
+:class:`iris.Future` definition.
+See :ref:`Usage of iris.FUTURE <iris_future_usage>`, below.
+
+
+.. _iris_api:
+
+Terminology : API, features, usages and behaviours
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The API is the components of the iris module and submodules which are
+"public" :  In Python, by convention, this means everything whose name does not
+have a leading underscore "_".
+This means all public modules and submodules, and their contained classes,
+public data and properties, functions and methods.
+
+A "feature" of the API includes public objects as above, but may also be used
+more loosely to indicate a class or mode of behaviour, for example when a
+keyword has a specific value, like "interpolate(mode='linear')".
+
+A "usage" is any code referring to public API elements, for example :
+
+        * `print iris.thing`
+        * `iris.submodule.call(arg1)`
+        * `iris.module.call(arg1, arg2, *more_args)`
+        * `iris.module.call(arg1, arg2, control=3)`
+        * `x = iris.module.class(arg, key=4)`
+
+A "behaviour" is whatever happens when you invoke a particular API usage,
+encompassing both returned values and any side effects such as code state
+changes or data written to files.
+
+The above examples are all public feature usages, and therefore should
+continue to work, with the same behaviours, at least until the next **major**
+version release.
+
+
+
+.. _iris_change_releases:
+
+Releases and Versions
+---------------------
+
+
+Iris releases have a unique identifying version string, in the form
+"<major>.<minor>.<micro><extension>", available to code as
+:data:`iris.__version__` .
+
+This contains major and minor release numbers.  The numbering and meaning of
+these are defined, following the `SemVer project <http://semver.org/>`_.
+
+The essential aspects of the "<major>.<minor>.<micro><extension>" arrangement
+are :
+
+    * "<major>", "<minor>" and "<micro>" are all integers, thus version
+      2.12 is later than 2.2 (i.e. it is "two point twelve", not "two point one
+      two").
+
+    * "<major>.<minor>" denote the software release version.
+
+    * A non-zero "<micro>" denotes a bugfix version, thus a release "X.Y.0" may
+      be followed by "X.Y.1", "X.Y.2" etc, which *only* differ by containing
+      bugfixes.  Any bugfix release supercedes its predecessors, and does not
+      change any (valid) APIs or behaviour :  there should be no reason not to
+      replace a given version with its latest bugfix successor.
+
+    * "<extension>" is blank for formal releases.  It used to indicate
```

Comment:
This section might need some tweaks to follow PEP440, but that might be a discussion for a different place. I don't think what's defined here would pass PEP440, though it might only result in a warning from `pip` at this point.